install-panel.sh

```diff
@@ -590,16 +590,43 @@ function firewall_firewalld {
   echo "* Firewall-cmd installed"
   print_brake 70
 }
+function centos_based_letsencrypt {
+    if [ "$OS_VER_MAJOR" == "7" ]; then
+      # Enable EPEL folowing official guide https://fedoraproject.org/wiki/EPEL
+      subscription-manager repos --enable "rhel-*-optional-rpms" --enable "rhel-*-extras-rpms"  --enable "rhel-ha-for-rhel-*-server-rpms"
+
+      # Install certbot 
+      yum install certbot
+    elif [ "$OS_VER_MAJOR" == "8" ]; then
+      # Enable EPEL folowing official guide https://fedoraproject.org/wiki/EPEL
+      ARCH=$(/bin/arch)
```

Comment:
Why is `ARCH` defined?

Reply:
I asked about that in discord you never answered :( 
I would say that we can just hardcode `x86_64` and call it a day. Of Course we can also define it in the variables section :)

Reply:
> I asked about that in discord you never answered :( 
> I would say that we can just hardcode `x86_64` and call it a day. Of Course we can also define it in the variables section :)

I saw your question on the Discord, my question here is why defining the variable `ARCH` Iis required in the first place?

Reply:
The `subscription-manager repos --enable "codeready-builder-for-rhel-8-${ARCH}-rpms"` needs architecture. I would just use x86_64.

Reply:
> The `subscription-manager repos --enable "codeready-builder-for-rhel-8-${ARCH}-rpms"` needs architecture. I would just use x86_64.

Yes, I understand. I find it overly complicated to activate `epel-release` in this way. Does this not work on CentOS 8?

```bash
sudo dnf install epel-release
sudo dnf install certbot
```

Is there any particular reason for not using the "built-in" epel-release?

Reply:
Ok let me start from very very start. So at first I had EPEP release downloaded by certbot start. Then I removed it bcs it was already installed before. Then here it says that you need to enable any recommended repositories. 
![image](https://user-images.githubusercontent.com/26341983/89764651-7acc1e80-dafd-11ea-9f66-9fed4b959a90.png)

So in [https://fedoraproject.org/wiki/EPEL#Quickstart](https://fedoraproject.org/wiki/EPEL#Quickstart) it says that some packages may require this 
![image](https://user-images.githubusercontent.com/26341983/89764746-ac44ea00-dafd-11ea-9f40-572c0361906d.png)

So following the certbot instructions to enable any recommendations I added that to the script. I don't know if the digitalocean page is correct I just read the instructions on the official certbots page. 

[https://certbot.eff.org/lets-encrypt/centosrhel7-other](https://certbot.eff.org/lets-encrypt/centosrhel7-other)
[https://certbot.eff.org/lets-encrypt/centosrhel8-other](https://certbot.eff.org/lets-encrypt/centosrhel8-other)

PS: it may very well work without them. I just followed the instructions.

Reply:
> Ok let me start from very very start. So at first I had EPEP release downloaded by certbot start. Then I removed it bcs it was already installed before. Then here it says that you need to enable any recommended repositories.
> ![image](https://user-images.githubusercontent.com/26341983/89764651-7acc1e80-dafd-11ea-9f66-9fed4b959a90.png)
> 
> So in https://fedoraproject.org/wiki/EPEL#Quickstart it says that some packages may require this
> ![image](https://user-images.githubusercontent.com/26341983/89764746-ac44ea00-dafd-11ea-9f40-572c0361906d.png)
> 
> So following the certbot instructions to enable any recommendations I added that to the script. I don't know if the digitalocean page is correct I just read the instructions on the official certbots page.
> 
> https://certbot.eff.org/lets-encrypt/centosrhel7-other
> https://certbot.eff.org/lets-encrypt/centosrhel8-other
> 
> PS: it may very well work without them. I just followed the instructions.

I understand. As far as I know, most of it is only for certain packages that may require it. Can you try without it and if it does not work, then add them?

CentOS 7:
```bash
sudo yum install certbot python2-certbot-nginx
```

CentOS 8:
```bash
sudo dnf install certbot python3-certbot-nginx
```

I am aware of the fact that the certbot documentation recommends enabling the recommended repositories, but I would like to avoid a conflict with the already installed `epel-release` (and I think certbot can do without the extra stuff).